reference-sheets/GitPractices.md

```diff
@@ -0,0 +1,204 @@
+
+# Usage
+
+Looks like you got through all the conceptualization. Nice job! However, that means absolutely nothing if you don't know how to put it into practice. So, in this section, we will go over the actual git command line inputs.
+
+Opening the command line isn't a difficult task. Visual Studio Code has one built in (accessible by pressing ctrl+\`), which makes using git much easier. 
+If you're not using VSCode, 
+Windows: Press *Win + X* to open the Command Prompt.
+Mac: Press *Cmd + Space* to open spotlight search, then type "Terminal" to open the Terminal.
+Linux: Spin around five times and then sing Mary Had A Little Lamb.
+
+### Cloning
+
+There are a few ways to clone a Github repository onto your computer, but the best way and the way that we'll be going over is by using the command line. 
+
+Go onto the GitHub website, then find the repository. There should be a green button that says "Code". Click that, and then copy the link that shows up under it. This will be used in the command. It tells git what exactly to clone off of the internet.
+
+Next, navigate using the command line to the target folder where you want your cloned repository to be on your computer. You can alternatively move the cloned folder after your clone it, but moving around using the command line makes me feel like a hacker so i do that instead.
+
+Type this text into the command line:
+
+`git clone <copied link>`
+
+It should look something like this:
+
+`git clone https://github.com/SciBorgs/SciGuides.git`
+
+This would clone this repository, the SciBorgs' SciGuides onto your computer in your working directory.
+
+### Branching
+
+To make a branch, type into the command line:
+
+`git branch <branch name>`
+
+Make sure that your chosen branch name follows the naming standards! Your name should be:
+1. Relevant: something indicative of what the branch is for
+2. All lowercase
+3. Spaces replaced with dashes
+
+So, these would be fine:
+- drivetrain
+- auto-align
+- feeder-fix
+
+But these wouldn't:
+- Drivetrain
+- auto_align
+- i-hate-git 
+
+However, after you decide to create the branch, you still need to switch to that branch!
+To do that, type into the command line:
+
+`git checkout <branch name>`
+
+You can also switch to a branch that isn't yours using `checkout`. If your friend was coding something and told you to go to their branch, you could just use checkout and it would work.
+
+### Committing
+
+Remember when we talked about staging our changes before committing? Well, to do that, type into the command line:
+
+`git add .`
+
+This will stage all of your working changes and make them ready for committing. The "." in this situation means all of your working changes.
+
+Of course, there is a way for you to stage only certain files but there's really no situation where it would be used.
```

Comment:
Not true. Please specify the method.

Also, you might want to mention `git status` somewhere in this section. It tells you what working changes have been added to the commit and what hasn't. Useful for double-checking and not having to revert.